src/app/component/eco-news/create-news/create-news.component.css

```diff
@@ -182,27 +182,75 @@ button {
     color: red;
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/80973503-20e5f980-8e28-11ea-9958-4641d7ffb78e.png)

Reply:
Please, fix layout according to the mockup(breakpoint 320px)

Reply:
fixed